pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -625,32 +637,38 @@ var _ = Describe("Converter", func() {
 				Serial: "e4686d2c-6e8d-4335-b8fd-81bee22f4815",
 			}
 
-			gracePerod := int64(5)
-			vmi.Spec.TerminationGracePeriodSeconds = &gracePerod
+			vmi.Spec.TerminationGracePeriodSeconds = pointer.P(int64(5))
 
 			vmi.ObjectMeta.UID = "f4686d2c-6e8d-4335-b8fd-81bee22f4814"
 		})
 
 		var convertedDomain = strings.TrimSpace(fmt.Sprintf(embedDomainTemplateX86_64, domainType, "%s"))
-		var convertedDomainWith5Period = fmt.Sprintf(convertedDomain, argMemBalloon5period)
-		var convertedDomainWith0Period = fmt.Sprintf(convertedDomain, argMemBalloon0period)
+		var convertedDomainWith5Period = fmt.Sprintf(convertedDomain, memBalloonWithModel(argMemBalloon5periodFmt, "virtio-non-transitional"))
+		var convertedDomainWith0Period = fmt.Sprintf(convertedDomain, memBalloonWithModel(argMemBalloon0periodFmt, "virtio-non-transitional"))
 		var convertedDomainWithFalseAutoattach = fmt.Sprintf(convertedDomain, argNoMemBalloon)
 
-		convertedDomain = fmt.Sprintf(convertedDomain, argMemBalloon10period)
+		convertedDomain = fmt.Sprintf(convertedDomain, memBalloonWithModel(argMemBalloon10periodFmt, "virtio-non-transitional"))
 
 		var convertedDomainppc64le = strings.TrimSpace(fmt.Sprintf(embedDomainTemplatePPC64le, domainType, "%s"))
-		var convertedDomainppc64leWith5Period = fmt.Sprintf(convertedDomainppc64le, argMemBalloon5period)
-		var convertedDomainppc64leWith0Period = fmt.Sprintf(convertedDomainppc64le, argMemBalloon0period)
+		var convertedDomainppc64leWith5Period = fmt.Sprintf(convertedDomainppc64le, memBalloonWithModel(argMemBalloon5periodFmt, "virtio-non-transitional"))
+		var convertedDomainppc64leWith0Period = fmt.Sprintf(convertedDomainppc64le, memBalloonWithModel(argMemBalloon0periodFmt, "virtio-non-transitional"))
 		var convertedDomainppc64leWithFalseAutoattach = fmt.Sprintf(convertedDomainppc64le, argNoMemBalloon)
 
-		convertedDomainppc64le = fmt.Sprintf(convertedDomainppc64le, argMemBalloon10period)
+		convertedDomainppc64le = fmt.Sprintf(convertedDomainppc64le, memBalloonWithModel(argMemBalloon10periodFmt, "virtio-non-transitional"))
```

Comment:
Camelcase?

Reply:
I would leave this for future improvements. I think these could be entirely re-written but I would avoid this PR increasing its size too much.